supertokens_python/recipe/session/interfaces.py

```diff
@@ -156,8 +156,8 @@ def get_global_claim_validators(
     @abstractmethod
     async def get_session(
         self,
-        access_token: str,
-        anti_csrf_token: Optional[str],
+        access_token: Optional[str],
+        anti_csrf_token: Optional[str] = None,
```

Comment:
@porcellus  `anti_csrf_token` having a default value (i.e. no need to pass one) is okay, right?

Reply:
it is